packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman.vue

```diff
@@ -27,15 +27,15 @@
 					<div class="section-row timespan">
 						<div class="button-column">
```

Comment:
Remove classes since it's not here anymore

Reply:
done!